dashboard/app/models/pd/application/teacher_application.rb

```diff
@@ -364,7 +364,7 @@ def workshop_present_if_required_for_status
     end
 
     def should_send_decision_email?
-      if regional_partner&.applications_decision_emails == RegionalPartner::SENT_BY_PARTNER
+      if regional_partner.nil? || regional_partner&.applications_decision_emails == RegionalPartner::SENT_BY_PARTNER
```

Comment:
I don't think the original check would work. Originally, we return `false` if the regional partner's settings are to send by partner. If there is no regional partner, we'd go to the `else` because the RP's settings are *not* send by partner.

But the nil check is redundant –– I've removed that!